### DISCUSSION TITLE
Mobile startup screen ad and other possible ways for collaboration (was: Meeting Notes: GARD & ASA Stats about the mobile startup screen ad and other possible ways for collaboration)

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#323] [Meeting]
Rylie from GARD and I have got an initial meeting directly initiated by [our Twitter announcement](https://twitter.com/asastatscom/status/1581277617184915456) for our mobile app sponsorship plans.

Ryle expressed GARD's intention to be presented in our mobile app as a way to attract more users and bring more awareness to their dApp. He was interested in costs and other conditions for their sponsorship and related mobile app's startup screen ad.

Rylie asked about our user base and I talked about historical data and brought up some thoughts about the future.

I have explained that based on many reasons we have decided that the first step should be their proposal to us. As an example, I have mentioned that it has been proven in the past that our community members aren't really eager to see the ASA Stats connects with the ecosystem's gambling dApps, and so we will refuse an offer from one similar project that reached us recently even if they offer happens to be much bigger than GARD's.

So I have emphasized that we'll pick a sponsor not just based on this mobile app sponsorship and that we are open to many other aspects and ways of collaboration. For example, as GARD has got a homonymous token, we can maybe start to reward our community members' contributions in GARD on top of the ASASTATS. Based on their offer, we can even open up additional sponsorship/ad space on our website too, etc.

We also discussed about ASA Stats' needs for the bridge/oracle network in preparation for our Voting dApp.

So we concluded that I should send some more reading material about our project in order for Rylie and the guys to get a grasp of our project. It is expected that the rest of our communication about this (and other things() should be email communication, so as a first step I'm sending those reading materials.

**Comment by ipaleka** [comment]
Follow-ups:

- - - - -
Hi Rylie!

Many people emphasized that our roadmap is unusually detailed and comprehensive reading material for a crypto project, so I guess that can be the starting point for you, at least for skimming it:

https://github.com/asastats/docs/blob/main/roadmap.md

The draft of our utilization document is published a few months ago:

https://github.com/asastats/channel/wiki/ASA-Stats-Token-utilization

Our mobile app development started as another public discussion:

https://github.com/asastats/channel/discussions/180

At the end of last year, we decided that DAO should completely take over the project in two years. Here's the very first announcement regarding the governors' selection:

https://github.com/asastats/dao-discussion/discussions/1

Bear in mind that your eventual sponsorship automatically implies that at least two members of your team (I guess you and David) will become ASA Stats DAO governors. :)

And here's that Voting dApp initial discussion:

https://github.com/asastats/dao-discussion/discussions/10

I'm looking forward to answering additional questions and/or suggestions from you!

Ivica

P.S. here are our meeting notes:

https://github.com/asastats/channel/discussions/323

**Reply by ipaleka** [reply]
- - - - -

Hi Ivica,

It was a pleasure chatting with you today. We're keen on putting out a proposal and given that you post everything publicly in your Github I figured we can just send something via email for you to relay to the community. I don't fully understand how we would be involved in governing ASA Stats as you mentioned above, however, we can at least get a proposal going.

To begin a partnership with ASA Stats and to help support your App's launch/DAO treasury we can create a unique referral link where both parties may track unique referrals to our platform. Per unique referral, we'd be willing to pay .05 GARD and we could run this program for two months. For now, we'd like to cap the amount we spend at 1000 GARD (20,000 referrals), however, depending on conversions we could consider keeping this going longer or paying out more than 1000 GARD total. Another alternative would be paying out a higher amount for users who actually take specific actions on the platform but not paying just for clicks.

It would be great to get some feedback from you/your community and we can go from there.

I hope we can work something out that is mutually beneficial!

Best reGARDs,

Rylie

- - - - -

**Reply by ipaleka** [reply]
- - - - -

Hi Rylie!

Thank you for your proposal! :)

It is added as a comment to the same discussion thread on our GitHub and our community members will discuss it there and in our other community channels (in which case we copy/paste valuable comments to the main thread on GitHub). When we'd have something more to ask or add, I'm going to reach you here by email.

Regarding ASA Stats DAO governor seats, basically, you don't need to do anything in the next 12 months. After that period, when DAO takes over the project, you'd be deciding about major DAO issues by using our Voting dApp.

Sincerely,

Ivica

- - - - -

**Reply by ipaleka** [reply]
- - - - -
By kerrilija in [Discord](https://discord.com/channels/906917846754418770/914540797054025788/1038767861017104426):

For eventual 20,000 referral clicks we get 1000$. What could we do with those funds?  From their perspective it's a good value, what about ours?

Is there a non-gambling, better offer?

- - - - -
By ipaleka

The funds from the sponsorship of mobile app are meant to be used to return to the devpool the funds borrowed by the admpool. No other concrete offers are received so far.
- - - - -

**Reply by ipaleka** [reply]
- - - - -
By kerrilija

Then it's logical that we accept the offer. What's your opinion?
- - - - -

**Reply by ipaleka** [reply]
> Then it's logical that we accept the offer. What's your opinion?

Let's talk about the scale of this project first. There's one year of heavy development and community contributions behind us where we have absolutely managed to keep up with all the new Algorand projects hitting the Mainnet. We have another year in front of us where pretty much all of our roadmap points can fit and get resources for successful deployments.

Not just that, we will publish a mobile app using our API this week and we're planning to deploy the Permission dApp on Mainnet - none of those have even taken place in our roadmap!

This project has managed to sustain an absolutely independent nature - the development so far has been funded only by our community investors. It's not that we planned to rely on that, we had absolutely tried hard to get a grant from the Algorand Foundation. Still, their fucked up policy so far has been utterly blind to the independent projects' efforts and contributions to the ecosystem. We really believe that the new grant system will change that.

That being said, my opinion is that US$1.000 worth of tokens isn't a good enough motive to change our pace and perception of our users, especially when our users' behavior tracking is on the table.

**Reply by kerrilija** [reply]
You are right actually. In terms of what you can do with 1000$ is not much. Let's try to look at things from a different perspective. How much is the advertising on our mobile app worth in a month? Hundreds of thousands of users that start the app will see that banner, so it definitely is an effective marketing move for an Algorand based project/company.

Maybe we should offer this service in more places (top projects' discord, tg, twitter msg), but we are kinda tight with time on that.

Users behaviour is definitely something to discuss before eventually accepting GARD's offer. Crypto's biggest values are decentralization and anonymity.

**Reply by ipaleka** [reply]
By motuwagon in [Discord](https://discord.com/channels/906917846754418770/914540797054025788/1038835765054021792):

If the only offer  for 1000 does not feel worth changing.  Would we gain more from giving the ad space to community engaged projects.  Who will then advertise to their group about the app.

- - - - -

**Reply by ipaleka** [reply]
> If the only offer for 1000 does not feel worth changing. Would we gain more from giving the ad space to community engaged projects. Who will then advertise to their group about the app.

As of now, we're going to craft a message and information that the mobile app is a gift for our users for the First Anniversary of ASA Stats.

**Reply by ipaleka** [reply]
Btw, this offer from GARD has been very valuable to us and we really appreciate it. We'll surely try to collaborate with them in the future in some other aspects, it's just that targeting 1st anniversary for us is plan B that suits us better than plan A involving tracking our users in a way.

**Comment by ipaleka** [comment]
By @kerrilija in [Discord](https://discord.com/channels/906917846754418770/914540797054025788/1039503648268492910):

Meanwhile, if a good marketing opportunity appears, we should consider it. Should we contact some other projects if they are interested?

- - - - -
ipaleka

The thing is that we don't have anything defined regarding marketing, there's no role nor fiat currency funds on our side to prepare and conduct such an initiative, respectively to reach influencers or other Algorand projects.
On the other hand, we can allocate dozens of millions of ASASTATS for the purpose, it's just that I don't have resources to plan and conduct anything.
Suggest something, organize a discussion about it here, including a needed budget and then ping me to allocate the funds.
In the future, that should be my communication with the community.

- - - - -
kerrilija — Today at 12:56 PM

This, regarding the advertisement on the welcome screen of our iOS and Android apps. In a similar manner that GARD contacted us, we could offer advertisement(banner with link) by sending twitter/discord DM or email messages to various projects. If we don't want to track our user behaviour, we can simply set a negotiable price for a period that works for us. That would eliminate the need for pay-per-click mechanism. We don't actually have to move any funds for that, IMO. Just by sending messages, making offers, it could actually bring revenue to ASA Stats. 

For example: 

Cosmic Champs have a beta version of the game up and running.

Headline launched Silo. 

Alchemon wants to bring new users.

We could send individual offers to each of them and see the outcome.

Our Twitter announcement regarding this might have been overlooked by some people that would want to be advertised by us.

- - - - -
ipaleka — Today at 1:59 PM

> This, regarding the advertisement on the welcome screen of our iOS and Android apps. In a similar manner that GARD contacted us, we could offer advertisement(banner with link) by sending twitter/discord DM or email messages to various projects. If we don't want to track our user behaviour, we can simply set a negotiable price for a period that works for us. That would eliminate the need for pay-per-click mechanism. We don't actually have to move any funds for that, IMO. Just by sending messages, making offers, it could actually bring revenue to ASA Stats.   For example:   Cosmic Champs have a beta version of the game up and running.  Headline launched Silo.   Alchemon wants to bring new users.  We could send individual offers to each of them and see the outcome.

We don't actually have to move any funds for that, IMO.

I guess some parts of a newly pinned message have been a cause for this sentence, so I should make that part clear: reaching out to people/projects is a job and jobs require payments. If I do that then that would have been part my overall job in ASA Stats. But, I don't have time for anything that's not directly related to development.

I'm against spamming other people/projects, but if the community decides that's the way then that would be our way. (edited)

- - - - -
kerrilija — Today at 2:02 PM

We need more feedback on that suggestion.

- - - - -
Maru the Cat — Today at 2:18 PM

In a way the Asa stats page in itself can be like the way to reach projects if they want to be a sponsor.  Like the splash screen or even inside the settings.  Maybe it could be like another link or area for sponsors with more info on how to be featured in Asa stats?  I’m sure most of the projects in ALGO already use Asa stats often and would see the opportunity if they saw in the site

- - - - -
kerrilija — Today at 6:16 PM

There should be no ads in the app, ever. Splash screen for a few seconds is ok. I don't really get what you're saying in the first sentence. Could you rephrase it? (edited)

- - - - -
Maru the Cat — Today at 6:22 PM

I was thinking that most projects in ALGO probably already use Asa stats
But a small section in the settings
For say sponsors
Or maybe just a way for projects to learn about how to become a sponsor
Within the app
Without putting an add in Asa stats
Not sure if that’s makes sense
I agree with no ads
The clean UX/UI of Asa stats is it’s strength
Ads would go against this

![image0](https://user-images.githubusercontent.com/49662536/200667064-0dd89a0e-f51b-4f45-8ed4-34883499da2d.jpg)

Maybe somewhere in here?

**Reply by ipaleka** [reply]
- - - - -
SCN9A — Today at 7:35 PM

I think the splash screen logo would be of interest to many projects. 

Our Tweet on October 15th was ideal, building excitement and anticipation. 
https://twitter.com/asastatscom/status/1581277617184915456

At this point, prior to mobile apps release, we need to work on three keys points and I really think an explainer/intro video would be ideal and suitable to share on social media. 

1- Selling point (elevator pitch).
As per tweet > We expect at least 1 million downloads within that time

I think we need to show some numbers, a kind of graph showing the growth in user-base. 

A:  Could be dynamic or static, one or more photo if necessary
B: Metric, any of the following and please suggest alternatives
    * General traffic / Unique visitors
    * Daily active users DAU, weekly active users WAU, monthly active users MAU
    * Number of fetched accounts

It is a logical to show proof of growth and large user base even though the logo is limited to mobile splash screen and the user-base (Mobile/web) would not be identical.

2- Setting expectations
From a project (prospective customer) perspective, I would like to know how to get featured and what does it entail? 
So we need to facilitate access to such information. 

A: are we open to extend featured logo to NFT projects? An Idea I support.
B: Is sponsorship fixed, custom? For example
* Fixed: https://www.nftexplorer.app/featured-slot fixed duration, winner bid gets the pre-defined slot?
* Custom: Vestige where projects have to email sales to get quota? negotiate duration/cost.

As per tweet
If you wish to appear on the app's startup screen for the first two months, get in touch with us

I presume one sponsor for 2 month is the plan. I’d like to propose an alternative, a fluid plan if you will, based on the following line of thought.

3- Call to action: say a project is sold on this, given scenarios:

A: if a particular project got a deal, how can we facilitate future deals with other projects? Waiting time is not ideal in a fast-paces ecosystem.
B: If we reach out to multiple projects and get a positive response, who shall get the spot? 

This leads to the fuild plan, where multiple projects can be featured in the same duration as explained below:
1- Screen logo: one logo at any time. 
2- Frequency of particular logo can be further discussed based on price and some other ideas for future development such as wallet holding. For now, a set price could be simple to implement, a set number of impressions in a given duration.

**Comment by ipaleka** [comment]
- - - - -
By Rylie in an email:

Thanks for the updates, I'll stay tuned to hear more from you!

- - - - -

Hi Rylie!

Your proposal initiated a broad discussion in our community and I'm so grateful to you for it.

https://github.com/asastats/channel/discussions/323

Nevertheless, as of now, it seems that we won't accept it in the first phase when our app hits the app stores. I'll be completely honest with you, just as I was in that linked discussion, the problem is in a kind of a paradigm shift for the way how we treat our users.

For the last year, we have been a reliable source on a regular basis for dozens of thousands of Algorand users and we have invested a lot in the development. At the same time, no user has been exposed in any mean to anybody during that time.

My opinion is that we shouldn't change that in the case of our mobile app where your requirement is that we track our users' behavior in a way. We are not dumb and we know that rarely any business can survive without implementing such a praxis in its business model, but your offer in relation to the scope of our project simply isn't good enough for us to take that step. 

Our discussion is still ongoing and there could be changes, of course, and also we're looking forward to collaborating in future subprojects (like for our user widgets system as I was talking to you in our meeting), so please stay tuned! :)

Best reGARDs,

Ivica

**Reply by kerrilija** [reply]
How about offering GARD a fixed price for the banner for 2 months? Regardless of the number of clicks.

**Reply by ipaleka** [reply]
> How about offering GARD a fixed price for the banner for 2 months? Regardless of the number of clicks.

The following from the OP is a starting point:
> I have explained that based on many reasons we have decided that the first step should be their proposal to us.

They gave us a capped proposal involving user tracking with another sub-condition for extending a cap if tracked users bring enough profit for them. All of that implies that they are most interested in exactly that way of collaboration regarding our mobile app.

Therefore, if we're going to offer that what you propose, the amount should be around the one they proposed and it's expected that it won't satisfy them. My opinion is that the scope of this project, regarding people's time, code developed, user base, and other aspects, shouldn't define a US$1,000 deal as a sponsorship. If we decide to shuffle various projects and their links as a pure advertisement in that startup screen, then that amount should probably be our offer. But, we need to have at least half of dozen of prospects for that.

**Reply by kerrilija** [reply]
That makes sense. Anyway, we are yet to hear their latest response. ASA Stats users are the perfect audience for an advertisement of such a project, or any ASA tbh.
